nntrainer/graph/network_graph.cpp

```diff
@@ -696,7 +702,7 @@ bool NetworkGraph::canExecuteInPlace(const std::shared_ptr<LayerNode> &lnode) {
    * 2. if the layer is not supporting backwarding, there is no dependency
    * requirement with other nodes for backwarding.
    */
-  if (no_op(lnode) || !lnode->supportBackwarding())
+  if (no_op(lnode) || no_op_shared(lnode) || !lnode->supportBackwarding())
```

Comment:
need to update the comment. 1, 2, and 3. Need to update no-op with shared.

Reply:
done.